src/amplitude-client.js

```diff
@@ -766,6 +772,10 @@ AmplitudeClient.prototype.saveEvents = function saveEvents() {
  * @example amplitudeClient.setDomain('.amplitude.com');
  */
 AmplitudeClient.prototype.setDomain = function setDomain(domain) {
+  if (this._shouldDeferCall()) {
+    return this._q.push(['setDomain', domain]);
```

Comment:
is it weird to return the result of array.push? or are you trying to shorthand
```javascript
{
   this._q.push...
   return
}
```

Reply:
shorthand. im doing this for the `set__`  where it's not expecting anything in return